.pre-commit-config.yaml

```diff
@@ -51,10 +51,12 @@ repos:
     rev: 1.7.0
     hooks:
       - id: bandit
+        language_version: python3
+        exclude: ^test/
```

Comment:
@asottile is there anything here we are doing wrong in relation with pre-commit ? I was following this solution https://github.com/PyCQA/bandit/issues/573#issuecomment-586347537

Reply:
UPDATE2: @asottile seems that pre-commit doesn't like long name arguments - it works with `-s` and crashing `--skip`. Is that possible ?

Reply:
it's because you had `--recursive .` which bypassed the filenames passed and was always wastefully linting everything (sometimes twice!)